### PR TITLE
Dispensers can obey gravity

### PR DESCRIPTION
**Pull request by weluvgoatz**
This adds a small option for dispensers to obey gravity.  With this, you can have moving dispensers by toggling gravity and putting them on a moving platform.  Enjoy.

### CODE REVIEW

File: src/badguy/dispenser.cpp [hunk 1]
```diff
@@ -224,8 +226,13 @@ Dispenser::collision(GameObject& other, const CollisionHit& hit)
 }
 
 void
-Dispenser::active_update(float )
+Dispenser::active_update(float dt_sec)
 {
+  if (m_gravity)
+  {
+    BadGuy::active_update(dt_sec);
+    m_physic.enable_gravity(true);
```
Comment (tobbi) [inline src/badguy/dispenser.cpp:234]: Does `Badguy::active_update` disable gravity again?

Comment (weluvgoatz) [reply src/badguy/dispenser.cpp:234]: Yes, it does.

Comment (tobbi) [reply src/badguy/dispenser.cpp:234]: Yeah, I know, but is it necessary to call `enable_gravity` again after calling `active_update`?

Comment (weluvgoatz) [reply src/badguy/dispenser.cpp]: It appears not, so it's been removed.

**Pull request merged**